protected/views/csv/upload.php

```diff
@@ -1,20 +1,26 @@
-<?php
-$form = $this->beginWidget(
-    'CActiveForm',
-    array(
-        'id' => 'upload-form',
-        'action' => Yii::app()->createURL('csv/preview', array('context' => $context)),
-        'enableAjaxValidation' => false,
-        'htmlOptions' => array('enctype' => 'multipart/form-data'),
-    )
-);
-
-echo $form->fileField(new Csv(), 'csvFile');
-if (isset($errors) and $errors !== null) {
-    echo '<pre>';
-    var_dump($errors);
-    echo '</pre>';
-}
-echo CHtml::submitButton('Submit');
-$this->endWidget();
+<div class="box admin">
+  <div class="errorSummary">
+      <?php
+      if (isset($errors) and $errors !== null) {
+          echo '<pre>';
+          echo ArrayHelper::array_dump_html($errors);
```

Comment:
This is much better.

Reply:
I think it looks messier when the code goes to html from php and straight back to php